packages/frontend/src/components/Explorer/ExploreTree/TableTree/TableTreeSections.tsx

```diff
@@ -326,6 +329,20 @@ const TableTreeSections: FC<Props> = ({
                         <Tooltip label="Write back custom metrics">
                             <ActionIcon
                                 onClick={() => {
+                                    if (
+                                        projectUuid &&
+                                        user.data?.organizationUuid
+                                    ) {
+                                        track({
+                                            name: EventName.WRITE_BACK_FROM_CUSTOM_METRIC_HEADER_CLICKED,
+                                            properties: {
+                                                userId: user.data.userUuid,
+                                                projectId: projectUuid,
+                                                organizationId:
+                                                user.data.organizationUuid,
```

Comment:
optional , add number of custom metrics available ?

Reply:
done